### PR TITLE
Implemented cron job to clean stale records

### PR DESCRIPTION
**Pull request by Alankrita08**
This pull request implements a cron job using django-cron to clean stale records (older than 30 days) from the database. This functionality ensures the automatic removal of outdated or irrelevant records, enhancing database performance and maintaining data consistency. Automating this task minimizes manual intervention, contributing to system efficiency and scalability.

**Key Features:**
Added CleanStaleRecordsCronJob in tasks.py to identify and clean stale records.
Configured settings.py to integrate and activate the cron job.
Validated functionality through database changes and execution logs.

**Evidence of Testing:**
A document with screenshots is attached, demonstrating:
Initial database state
Stale record insertion
Cron job execution
Post-cron job database state
Cron job logs

[Evidence Document for Cron Job Implementatio1.pdf](https://github.com/user-attachments/files/17953307/Evidence.Document.for.Cron.Job.Implementatio1.pdf)

### CODE REVIEW

File: core/settings.py [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django_extensions",
+    'django_cron',
```
Comment (jbm1412) [inline core/settings.py:66]: Please add django_cron and its version in the requirements.txt file

**Review by jbm1412 — CHANGES_REQUESTED** [review]

File: core/settings.py (hunk 1 above)
Comment (Alankrita08) [reply core/settings.py:66]: @jbm1412 Added django-cron to requirements.txt, resolved merge conflicts, and ensured unnecessary log files are ignored. Let me know if any further changes are needed.
image

**Review by s-bonnor — CHANGES_REQUESTED** [review]
Looks good, just need you to fix the merge conflicts

**Comment by Alankrita08** [comment]
@s-bonnor fixed the merge conflicts, Thankyou

**Review by s-bonnor — APPROVED** [review]
Looks good

**Pull request merged**